src/components/TopicElements/TopicElements.js

```diff
@@ -156,12 +157,29 @@ function TopicElements({ history, appBaseUrl, staticFilesUrl }) {
 
   const appElements = getComponents(appComponents, elements);
 
+  const getScreenSize = (size) => {
+    let found = false;
+    Object.entries({
+      xs: 576,
+      s: 768,
+      m: 992,
+      l: 1200,
+      xl: Infinity,
+    }).forEach((brkpt) => {
+      if (!found && size <= brkpt[1]) {
+        // eslint-disable-next-line prefer-destructuring
+        found = brkpt[0];
+      }
+    });
+    return found;
```

Comment:
Since the ResizeHandler already calculate this you could modifiy the component to give this value in the onResize callback:
```
onResize(entries, widthBreakpoint, heightBreakpoint
```